pkg/controller/resourcequota/resource_quota_controller.go

```diff
@@ -120,7 +119,9 @@ func NewController(options *ControllerOptions) (*Controller, error) {
 
 	options.ResourceQuotaInformer.Informer().AddEventHandlerWithResyncPeriod(
 		cache.ResourceEventHandlerFuncs{
-			AddFunc: rq.addQuota,
+			AddFunc: func(obj interface{}) {
+				rq.addQuota(ctx, obj)
```

Comment:
All of these callbacks are always going to get the same ctx. Better look up the controller once here and pass around the controller.

/cancel lgtm

Reply:
/remove lgtm

Reply:
/lgtm cancel

Reply:
That's the magic incantation :)

Reply:
Updated, PTAL!